_posts/2022-03-04-Alternative-Install-Methods.md

```diff
@@ -21,6 +21,7 @@ Jamulus (mostly the GUI-Client, not the headless server) is included in the main
 * Jamulus is available in the [Debian repository](https://packages.debian.org/sid/sound/jamulus). It might be outdated, especially in Debian Stable. To install the client, just use apt: `sudo apt install jamulus`
 * Jamulus is available as [Flatpak on Flathub](https://flathub.org/apps/details/io.jamulus.Jamulus). You might need to set up Pipewire in order to get Jamulus to run.
 * Tormod Volden provides a [PPA for Ubuntu (and probably Debian) on launchpad.net](https://launchpad.net/~tormodvolden/+archive/ubuntu/jam/+packages)
+* Jamulus is available for archlinux in the [AUR repository](https://aur.archlinux.org/packages?O=0&K=jamulus). 3 versions are provided. jamulus is the regular one with client and server with GUI, jamulus-headless is the headless server version without GUI and jamulus-git, a VCS with the git actual state. If you dont want to build from source and prefer a binary repository, you can have a look to [osam-cologne repository](https://github.com/osam-cologne/archlinux-proaudio) and that you will be able to install jamulus with pacman.
```

Comment:
> If you dont want to build from source

Does this mean that
> Jamulus is available for archlinux
only as source?

(Not knowing ArchLinux, this isn't clear to me.  Maybe it's obvious to those who need to know, though.)

Reply:
In archlinux, there are three repositories with binaries. There is also an offcial users repository called AUR (archlinux users repository). When you want to install a package from AUR, the build is made locally but automatically on your computer.
It is long and needs a lot of cpu and HDD. That's why there are some other repositories from users that provides binaries. This is the case of osam-cologne repos, for all the pro-audio stuff that is not already in the official binary repository.
Is it clear now ?